core/src/main/scala/org/apache/spark/package.scala

```diff
@@ -76,7 +76,9 @@ package object spark {
         )
       } catch {
         case e: Exception =>
-          throw new SparkException("Error loading properties from spark-version-info.properties", e)
+      //  throw new SparkException("Error loading properties from spark-version-info.properties", e)
+          val unknownProp = "<unknown>"
+          (unknownProp, unknownProp, unknownProp, unknownProp, unknownProp, unknownProp)
```

Comment:
Is this safe change? Looks like behavior change when throwing exception.

Reply:
Thank you, this is my mistake. I will revert this.